test/infra/infra-dbdeployer-mysql84-gr/docker/entrypoint.sh

```
@@ -31,7 +31,8 @@ dbdeployer deploy replication "${MYSQL_VERSION}" \
     -c sync_binlog=0 \
     -c binlog_checksum=NONE \
     -c mysql_native_password=ON \
-    -c innodb_use_native_aio=0
+    -c innodb_use_native_aio=0 \
+    -c report_host=dbdeployer1.${INFRA}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)